doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -139,7 +139,7 @@ In older releases, internal level sizes mean that the DB can fully utilize only
 specific partition / LV sizes that correspond to sums of L0, L0+L1, L1+L2,
 etc. sizes, which with default settings means roughly 3 GB, 30 GB, 300 GB, and
 so forth.  Most deployments will not substantially benefit from sizing to
-accomodate L3 and higher, though DB compaction can be facilitated by doubling
+accommodate L3 and higher, though DB compaction can be facilitated by doubling
```

Comment:
:o

Reply:
:o indeed.

Reply:
That mistake was mine about a year and a half ago.  My ignominy is unbounded.

Reply:
This looks like a typo to me, so I think you're acquitted.

Here's ignominy: Until I was 27, I had to look up "recommend" to see whether it had one or two ms in it. That's ignominy.

Reply:
_Manoever_ remains my nemesis.

Reply:
I read this as "Maneater". Like Hall and Oates.